bps/bps_ApPipe.yaml

```diff
@@ -10,11 +10,25 @@ campaign: smallrun_example
 
```

Comment:
templateDataId is the same as the default.  I haven't yet heard of a use case that needed to change it.  I'd remove it until it actually needs to be different.